Added convert_csv2r function and additional files for this function.

piotrpatrzylas opened the pull request:
Convert_csv2r:
-takes .csv files with codes for merge_everything and creates global variables that can be used as an input for this function

Commit:
Added convert_csv2r function and additional files for this function.

simonthelwall commented on R/convert_csv2r.R:
```diff
@@ -0,0 +1,58 @@
+#' Convert .csv file to global variable that can be used in merge_everything function.
```
Could you add some more documentation and an example? I think you should describe what the .csv file should look like, how it should be structured and what its' purpose is.

simonthelwall commented on R/convert_csv2r.R:
```diff
@@ -0,0 +1,58 @@
+#' Creates ccg_list objects.
+#'
+#' @param ccg_file A file with CCG codes.
+#' @return R object(list).
```
Need more information about the structure of the list here

simonthelwall commented on R/convert_csv2r.R:
```diff
@@ -0,0 +1,58 @@
+#'
+#' Creates trust_list objects.
+#'
+#' @param trust_file A file with CCG codes.
```
The terminology needs to be tightened up here a trust is not a CCG and a CCG is not a trust.

simonthelwall reviewed (changes requested):
Hi Piotr, thanks for the contributions. I've made some comments on the files that I'd like to see addressed before the pull request is accepted. 

Regards,

Simon

Commit:
Updated documentation for convert_csv2r

simonthelwall reviewed (approved):
Thanks for making these changes. As discussed in person, I'll accept these changes into dev and make some changes before merging into master.

simonthelwall committed:
Merge branch 'dev' into tablemerge

simonthelwall committed:
Fixed Description file

TravisCI found an error in which imports was incorrectly formatted. Needed a comma after utils in
    utils
    RcppRoll
